bindings/python/iota_sdk/client/_node_indexer_api.py

```diff
@@ -95,20 +113,9 @@ class QueryParameters:
         timelocked_before: Optional[int] = None
         unlockable_by_address: Optional[str] = None
 
-    class OutputIdsResponse:
-        """Response type for output IDs.
-
-        Attributes:
-            ledger_index: The ledger index for which the response is valid.
-            cursor: The cursor to the next page of results.
-            items: The query results.
-        """
-
-        def __init__(self, output_dict: Dict):
-            self.ledgerIndex = output_dict["ledgerIndex"]
-            self.cursor = output_dict["cursor"]
-            self.items = [OutputId.from_string(
-                output_id) for output_id in output_dict["items"]]
+    @abstractmethod
+    def _call_method(self, name, data=None):
+        return {}
```

Comment:
What's this for?

Reply:
otehrwise all _call_method fns show as not found. thats why the abstract stuff.

Reply:
Otherwise: `iota_sdk/client/_node_indexer_api.py:126:19: E1101: Instance of 'NodeIndexerAPI' has no '_call_method' member (no-member)`

Reply:
Stupid linter